### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by davidbrochart** [mitosch/textual-terminal#3]
It could be nice to support Windows, maybe using [pywinpty](https://github.com/andfoy/pywinpty)?

**Comment by mitosch** [comment]
Thank you for your feedback. I will have a look into it and try to setup a windows box.

Which terminal do you suggest to use for windows? (Besides cmd.exe :-)

**Comment by davidbrochart** [comment]
I only know `cmd.exe`.

**Comment by mitosch** [comment]
I managed to install a Windows Python environment. The following packages are unix specific: fcntl, termios, pty.

**Comment by davidbrochart** [comment]
Yes, you can have a look at [jupyverse](https://github.com/jupyter-server/jupyverse/blob/09870a3de4d1b688ae59ee41b63860409a3700a9/plugins/terminals/fps_terminals/win_server.py) for an example of using `winpty`.
